refinery/core/models.py

```diff
@@ -1730,6 +1729,12 @@ class AnalysisNodeConnection(models.Model):
                                            default=False)
     galaxy_dataset_name = models.CharField(null=True, blank=True,
                                            max_length=250)
+    # state from galaxy
+    state = models.CharField(null=True, blank=True, max_length=100)
+    # size of the file
+    file_size = models.IntegerField(null=True, blank=True)
+    # galaxy dataset id
+    dataset_id = models.CharField(null=True, blank=True, max_length=100)
```

Comment:
Are these really necessary? AnalysisNodeConnection already has a lot of fields that appear to be used only once (during analysis).

Reply:
we could change the return value of the function create_analysis_output_node_connections() https://github.com/refinery-platform/refinery-platform/pull/3461/files#diff-49577ffb85a5a9e95447df9c8ccc34eeR473 to be a dict that represents the objects of the filter operation on the AnalysisNodeConnection at https://github.com/refinery-platform/refinery-platform/pull/3461/files#diff-49577ffb85a5a9e95447df9c8ccc34eeR476.  Then we wouldn't have to store the data in the database for re-use, although it would make the create_analysis_output_node_connections() a little opaque in terms of what it does.

Reply:
I would say it's a better choice than making database schema even more complicated.